docs/config/ping-federate-example.md

```diff
@@ -9,87 +9,38 @@ Ensure you have installed the Identity Service before starting. You will also ne
-   * Choose *NONE* in the **SAML Signature Key Name** dropdown.
-
-7. After clicking **Save**, navigate to the **Mappers** tab and select **Create**.
-8. Make the following configurations:
-
-   * Enter *email* into **Name**.
-   * Choose *Attribute Importer* from the **Mapper Type** dropdown.
-   * Enter *EMAIL* into **Attribute Name**.
-   * Enter *email* into **User Attribute Name**.
-
-9. After clicking **Save** navigate back to the **Identity Provider Mappers** using the breadcrumb and select the **Export** tab.
-10. Click **Download** and make a note of the file name and download location.
-11. Create a file with the extension `.cert` (the file name isn't important other than to locate it later on) with the following content:
-
-    ```
-    -----BEGIN CERTIFICATE-----
-    -----END CERTIFICATE-----
-    ```
-
-12. Open the file you downloaded in Step 10 with a text editor.
-13. Find the line beginning with the tag `<dsig:X509Certificate>` and ending with `</dsig:X509Certificate>`.
-![X509 Certificate from export](./keycloak-export-cert.png)
-14. Copy the string in-between these tags and paste the content into the file you created in Step 11 in-between `-----BEGIN CERTIFICATE-----` and
-    `-----END CERTIFICATE-----`.
-    
-The contents of your `.cert` file should now look similar to the following:
+* Open a new text file and paste the following into it.
+```
+-----BEGIN CERTIFICATE-----
 
+-----END CERTIFICATE-----
+```
+* In a seperate tab, open the **certificate descriptor API** at **{BASE-URI}/auth/realms/{REALM-NAME}/protocol/saml/descriptor**.
+* In the middle of the two lines, add the value found between **<dsig:X509Certificate>** and **</dsig:X509Certificate>** in the **certificate descriptor API** such as in the following example.
 ```
 -----BEGIN CERTIFICATE-----
 MIICnzCCAYcCBgFkqEAQCDANBgkqhkiG9w0BAQsFADATMREwDwYDVQQDDAhhbGZyZXNjbzA
 -----END CERTIFICATE-----
 ```
+Save the file giving it a name that ends with **.cert**, in this example we will use **certificate.cert**.
 
-### 3. Configure your PingFederate connection
+### Configure your PingFederate connection
 
 1. Sign in to your PingFederate instance as a user with administrative privileges.
 2. Navigate to **SP Connections** and select **Create New**.
 3. Ensure the **Browser SSO Profiles** connection template with *Protocol SAML 2.0* is selected and then click **Next**.
 4. On the **Connection Options** tab ensure only **Browser SSO** is selected.
-5. On the **Import Metadata** tab select **Choose file** and find the file downloaded in Step 10 of the previous section. 
-6. On the **General Info** tab verify that the imported values correspond to those from the Identity Service setup.
+
+5. On the **General Info** tab 
+  * In **Parter's Entity Id** add the value given for **Location** in the **certificate descriptor API**, but remove everything after **{REALM-NAME}**.
+  * Add a connection name.
+  * In Base URL, add {Base-URL}.
```

Comment:
What is `{Base-URL}`? Is this a literal value? Or is this some value that is already known?